scripts/audit-i18n-coverage.sh

```
@@ -0,0 +1,288 @@
+
+total=0
+count_full=0
+count_partial=0
+count_none=0
+count_legacy=0
+
+while IFS= read -r -d '' file; do
+    total=$((total + 1))
+
+    # Path relative to project root
+    rel_path="${file#${PROJECT_ROOT}/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)